<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>924</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>851</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>839</width>
       <height>849</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>821</width>
        <height>821</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="actAddfiles"/>
    <addaction name="actDelList"/>
    <addaction name="actForeachList"/>
    <addaction name="actAddFolder"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actActSize"/>
    <addaction name="actZoomOut"/>
    <addaction name="actZoomIn"/>
    <addaction name="actFitHeight"/>
    <addaction name="actFFitWeight"/>
    <addaction name="actWindowsHide"/>
    <addaction name="actWindowsFloat"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddfiles"/>
   <addaction name="actForeachList"/>
   <addaction name="actDelList"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actActSize"/>
   <addaction name="actFitHeight"/>
   <addaction name="actFFitWeight"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actWindowsHide"/>
   <addaction name="actWindowsFloat"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>38</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>图片目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>781</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Item</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>type</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>new item</string>
      </property>
      <property name="text">
       <string>root</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <action name="actAddfiles">
   <property name="text">
    <string>添加文件</string>
   </property>
  </action>
  <action name="actAddFolder">
   <property name="text">
    <string>添加目录</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actForeachList">
   <property name="text">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actDelList">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actActSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
  </action>
  <action name="actFitHeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
  </action>
  <action name="actFFitWeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
  </action>
  <action name="actWindowsHide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
  </action>
  <action name="actWindowsFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
